### PR TITLE
Instructions to do CA rotation in CAPI

### PR DESCRIPTION
**Pull request by namnx228**

### CODE REVIEW

File: ca-rotation-script/CA-Rotation-CAPI.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+my-controlplane           true          1          1                1                  
+
+$ kubectl get machinedeployment
+NAME     PHASE     REPLICAS   AVAILABLE   READY
+worker   Running   1          1           1
+```
+## Pivoting
+If we want to make sure that the CA rotation procedure works with the case that the cluster is self-hosted, we have to do pivoting.
+
+Firstly, transform the workload cluster into another management cluster:
```
Comment (kashifest) [inline ca-rotation-script/CA-Rotation-CAPI.md:62]: This line is not clear. By management cluster what do you mean? Rather you can say, initialize the provider components on the target cluster?

File: ca-rotation-script/CA-Rotation-CAPI.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+```sh
+clusterctl init --kubeconfig capi-kubeconfig --infrastructure docker
+```
+Next, move all CAPI components from the original management cluster to the new management cluster:
```
Comment (kashifest) [inline ca-rotation-script/CA-Rotation-CAPI.md:66]: Is it moving only CAPI components or rather infrastructure-provider components also?

**Review by kashifest — COMMENTED** [review]
This looks good. Fer questions/inline.

File: ca-rotation-script/CA-Rotation-CAPI.md (hunk 2 above)
Comment (fmuyassarov) [inline ca-rotation-script/CA-Rotation-CAPI.md:66]:
```suggestion
Next, move all CAPI components from the original management cluster to the target cluster:
```

**Review by fmuyassarov — COMMENTED** [review]
Looks good!

File: ca-rotation-script/CA-Rotation-CAPI.md (hunk 1 above)
Comment (namnx228) [reply ca-rotation-script/CA-Rotation-CAPI.md:62]: True, that would be clearer. Thanks

**Review by furkatgofurov7 — COMMENTED** [review]
LGTM

File: ca-rotation-script/CA-Rotation-CAPI.md (hunk 2 above)
Comment (namnx228) [reply ca-rotation-script/CA-Rotation-CAPI.md]: Fixed! Thanks

**Review by fmuyassarov — APPROVED** [review]

**Pull request merged**